### PR TITLE
Add new swagger version to schema

### PR DESCRIPTION
**Pull request by leonardbf**
New swagger for RP API version 2019-05-01 has been created for GA.

### CODE REVIEW

**Comment by TianoMS** [comment]
> CodeFlow for GitHub does not download executable files. If you trust this file and would like to run it, please download it yourself through GitHub.

you may want to remove this file. 

---
Refers to: tests/2019-05-01/.DS_Store:1 in 9565a56. [](commit_id = 9565a5613989710d7b8410b83518b864195261cc, deletion_comment = False)

File: schemas/2019-05-01/Microsoft.NetApp.json [hunk 1]
```diff
@@ -0,0 +1,671 @@
+{
+  "id": "https://schema.management.azure.com/schemas/2019-05-01/Microsoft.NetApp.json#",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Microsoft.NetApp",
+  "description": "Microsoft NetApp Resource Types",
+  "resourceDefinitions": {
+    "netAppAccounts": {
```
Comment (TianoMS) [inline schemas/2019-05-01/Microsoft.NetApp.json]: >netAppAccounts [](start = 5, length = 14)

please make changes in the root templates:
2014-04-01-preview/deploymentTemplate.json, 
2015-01-01/deploymentTemplate.json,
2019-04-01/deploymentTemplate.json.

**Comment by leonardbf** [comment]
> > CodeFlow for GitHub does not download executable files. If you trust this file and would like to run it, please download it yourself through GitHub.
> 
> you may want to remove this file.
> 
> Refers to: tests/2019-05-01/.DS_Store:1 in [9565a56](https://github.com/Azure/azure-resource-manager-schemas/commit/9565a5613989710d7b8410b83518b864195261cc). [](commit_id = [9565a56](https://github.com/Azure/azure-resource-manager-schemas/commit/9565a5613989710d7b8410b83518b864195261cc), deletion_comment = False)

Ah one slipped through. Pesky mac files. Will remove.

File: schemas/2019-05-01/Microsoft.NetApp.json (hunk 1 above)
Comment (leonardbf) [reply schemas/2019-05-01/Microsoft.NetApp.json]: > 2014-04-01-preview/deploymentTemplate.json,
> 2015-01-01/deploymentTemplate.json,
> 2019-04-01/deploymentTemplate.json.

Ok, entries added.

**Comment by leonardbf** [comment]
@TianoMS Thanks for looking at this PR. I've made the changes however there are now failing checks. The new schema which I now reference in the deployment templates does not yet exist in that location. Is there something more I need to do?

File: schemas/2019-04-01/deploymentTemplate.json [hunk 2]
```diff
@@ -968,6 +968,10 @@
                   { "$ref": "https://schema.management.azure.com/schemas/2017-08-15/Microsoft.NetApp.json#/resourceDefinitions/capacityPools" },
                   { "$ref": "https://schema.management.azure.com/schemas/2017-08-15/Microsoft.NetApp.json#/resourceDefinitions/volumes" },
                   { "$ref": "https://schema.management.azure.com/schemas/2017-08-15/Microsoft.NetApp.json#/resourceDefinitions/snapshots" },
+                  { "$ref": "https://schema.management.azure.com/schemas/2019-05-01/Microsoft.NetApp.json#/resourceDefinitions/netAppAccounts" },
+                  { "$ref": "https://schema.management.azure.com/schemas/2019-05-01/Microsoft.NetApp.json#/resourceDefinitions/capacityPools" },
```
Comment (TianoMS) [inline schemas/2019-04-01/deploymentTemplate.json]: >capacityPools [](start = 127, length = 13)

I think this needs to be: netAppAccounts_capacityPools

File: schemas/2019-04-01/deploymentTemplate.json [hunk 3]
```diff
@@ -968,6 +968,10 @@
...
+                  { "$ref": "https://schema.management.azure.com/schemas/2019-05-01/Microsoft.NetApp.json#/resourceDefinitions/volumes" },
```
Comment (TianoMS) [inline schemas/2019-04-01/deploymentTemplate.json]: >volumes [](start = 127, length = 7)

netAppAccounts_capacityPools_volumes

File: schemas/2019-04-01/deploymentTemplate.json [hunk 4]
```diff
@@ -968,6 +968,10 @@
...
+                  { "$ref": "https://schema.management.azure.com/schemas/2019-05-01/Microsoft.NetApp.json#/resourceDefinitions/snapshots" },
```
Comment (TianoMS) [inline schemas/2019-04-01/deploymentTemplate.json]: >snapshots [](start = 127, length = 9)

netAppAccounts_capacityPools_volumes_snapshots

Comment (leonardbf) [reply schemas/2019-04-01/deploymentTemplate.json]: > > snapshots [](start = 127, length = 9)
> 
> netAppAccounts_capacityPools_volumes_snapshots

Yes of course. These were renamed in the latest version which was auto generated. Will fix.

**Comment by leonardbf** [comment]
@TianoMS Can this now be merged?

**Comment by leonardbf** [comment]
@TianoMS  Can this now be merged?

**Pull request merged**